Providers/Scripts/2.4x-2.5x/Scripts/Tests/test_nxProviders.py

```diff
@@ -30,8 +30,8 @@ def opened_w_error(filename, mode="r"):
 try:
     import unittest2
 except:
-    os.system('tar -zxvf ./unittest2-0.5.1.tar.gz ')
-    os.system('cd ./unittest2-0.5.1 && python' + sys.version.split()[0][:3]  + ' ./setup.py install --prefix=/usr/local')
+    os.system('tar -zxvf ./unittest2-0.5.1.tar.gz')
```

Comment:
Maybe check that the folder does not exist first to prevent having to extract it every time the unit tests are run.